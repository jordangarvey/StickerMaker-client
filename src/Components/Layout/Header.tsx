import React, { FC } from "react";
import { StyleSheet, View} from "react-native";

import * as C from "../../Global/Colours";


const Header: FC = () => <View style={styles.header}/>;

const styles = StyleSheet.create({
	header: {
		backgroundColor: C.secondary,
		borderColor: "#000000",
		borderWidth: 0.5,
		height: 100,
		width: "100%"
	}
});

export default Header;
