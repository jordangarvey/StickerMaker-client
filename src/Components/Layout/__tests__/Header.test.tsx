import React from "react";

import renderer from "react-test-renderer";

import Header from "../Header";


describe("The Header componentâ€™s snapshot", () => {
	it("renders as expected", () => {
		const component = renderer.create(
			<Header/>
		).toJSON();


		expect(component).toMatchSnapshot();
	});
});
