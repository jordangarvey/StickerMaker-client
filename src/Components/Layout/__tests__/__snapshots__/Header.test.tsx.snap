// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Header componentâ€™s snapshot renders as expected 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "borderColor": "#000000",
      "borderWidth": 0.5,
      "height": 100,
      "width": "100%",
    }
  }
/>
`;
